### PR TITLE
billing stuff Rel 701 for patch1

### PR DESCRIPTION
**Pull request by stephenwaite**
Fixes #

### CODE REVIEW

File: version.php [hunk 1]
```diff
@@ -28,7 +28,7 @@
 // is a database change in the course of development.  It is used
 // internally to determine when a database upgrade is needed.
 //
-$v_database = 486;
+$v_database = 487;
```
Comment (bradymiller) [inline version.php:31]: would not increment this on patch database updates since not sure what it will do to the admin.php stuff

Comment (bradymiller) [reply version.php:31]: (btw, i can revert this change if needed when i do the patch tonight/tomorrow). i still have one more minor fixup commit to do on my recent changes and I can revert it there. So feel free to just bring this in as is.

**Comment by bradymiller** [comment]
also there are some odd commits here (commits already in the codebase), so would bring this pr in as a "squash and merge" to ensure the odd commits don't make it into the codebase

File: version.php (hunk 1 above)
Comment (stephenwaite) [reply version.php]: followed your advice, should be all set now, thanks

**Pull request merged**